shared/src/common/types/user-details/user-details-response-dto.type.ts

```diff
@@ -0,0 +1,10 @@
+type UserDetailsResponseDto = {
+  id: number;
+  firstName: string;
+  lastName: string;
+  gender: string;
+  avatarUrl: string;
+  dateOfBirth: string;
```

Comment:
```suggestion
  firstName: string | null;
  lastName: string | null;
  gender: string | null;
  avatarUrl: string | null;
  dateOfBirth: string | null;
```
aren't they all nullable?

Reply:
@PavelSukhinin in get request I return found dto or null( if not found), so that's why I wrote these fields without null